src/Core/Enums/AuthRequestType.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+namespace Bit.Core.Enums
+{
+    public enum AuthRequestType : byte
+    {
+        AuthenticateAndUnlock = 0,
+        Unlock = 1,
```

Comment:
❓ Just in case, I see this one not being used currently. Is this alright?

Reply:
It is the same enum as we have on the server.

Reply:
it is just there for consistency so far.